src/services/salary/salaryService.ts

```
@@ -122,62 +125,63 @@ export async function processSalaryBatch(batchId: string): Promise<void> {
-
-      await prisma.salaryItem.update({
-        where: { id: item.id },
-        data: {
-          status: result.status,
-          transactionId: result.transactionId,
-          errorMessage:
-            result.status === "failed" ? "Transfer payment failed" : null,
-        },
-      });
+  const BATCH_CONCURRENCY = 10;
+  const pending: SalaryItem[] = (batch.items as SalaryItem[]).filter((item) => item.status !== "completed");
```

ESLint

[error] Replace `(item)·=>·item.status·!==·"completed"` with `⏎····(item)·=>·item.status·!==·"completed",⏎··`

(prettier/prettier)